packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -922,6 +922,27 @@ void main() {
 
   }, variant: TargetPlatformVariant.desktop());
 
+  // Regression test for https://github.com/flutter/flutter/issues/151878.
+  testWidgets('Searching for non matching item does not crash', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: Scaffold(
+        body: DropdownMenu<TestMenu>(
+          enableFilter: true,
+          dropdownMenuEntries: menuChildren,
+        ),
+      ),
+    ));
+
+    // Open the menu.
+    await tester.tap(find.byType(DropdownMenu<TestMenu>));
+    await tester.pump();
+    await tester.enterText(find.byType(TextField).first, 'Me');
+    await tester.pumpAndSettle();
+    await tester.enterText(find.byType(TextField).first, 'Meu');
+    await tester.pumpAndSettle();
+    expect(tester.takeException(), isNull);
+  }, variant: TargetPlatformVariant.desktop());
```

Comment:
We can remove this variant and set `requestFocusOnTap` to true

Reply:
Have done the change, please have a look.